src/Microsoft.AspNetCore.Mvc.Razor.Host/MvcRazorParser.cs

```diff
@@ -31,7 +30,7 @@ public class MvcRazorParser : RazorParser
         /// from parsed pages from _ViewImports files.</param>
         /// <param name="defaultInheritedChunks">The <see cref="IReadOnlyList{Chunk}"/> inherited by
         /// default by all Razor pages in the application.</param>
-        /// <param name="modelExpressionTypeName">The full name of the model expression <see cref="Type"/>.</param>
+        /// <param name="modelExpressionTypeName">The full name of the model expression <see cref="Type"/>.</param> 
```

Comment:
Not mandatory but remove trailing whitespace added here if you open the file again.